.storybook/storybook.scss

```diff
@@ -0,0 +1,25 @@
+@import '../src/sass/config';
+@import '../src/sass/mixins';
+@import '../src/components/buttons/buttons.stories';
+
+.sg-story {
```

Comment:
do we need this css? layout  and background of the story should be customised on story param level and handled in story block

Reply:
This styles are for util component to present story variant in consistent way. Not sure if we can do this with SB story blocks